mne/io/tests/test_meas_info.py

```diff
@@ -479,32 +513,18 @@ def test_anonymize(tmpdir):
                                     duration=[1, 1],
                                     description='dummy',
                                     orig_time=None))
-    raw.info['subject_info'] = dict(id=1, his_id='foobar', last_name='bar',
-                                    first_name='bar', birthday=(1987, 4, 8),
-                                    sex=0, hand=1)
-
-    # Test no error for incomplete info
-    info = raw.info.copy()
-    info.pop('file_id')
-    anonymize_info(info)
 
     # Test instance method
     events = read_events(event_name)
     epochs = Epochs(raw, events[:1], 2, 0., 0.1, baseline=None)
 
-    assert not any(_is_anonymous(raw))
-    raw.anonymize()
-    assert all(_is_anonymous(raw))
+    _test_anonymize_info(raw.info.copy())
+    _test_anonymize_info(epochs.info.copy())
 
-    assert not any(_is_anonymous(epochs)[:-1])  # epochs has no annotations
-    epochs.anonymize()
-    assert all(_is_anonymous(epochs)[:-1])
-
-    # When we write out with raw.save, these get overwritten with the
-    # new save time
-    out_fname = tmpdir.join('test_subj_info_raw.fif')
-    raw.save(out_fname, overwrite=True)
-    assert all(_is_anonymous(read_raw_fif(out_fname)))
+    # test that annotations are correctly zeroed
+    raw.anonymize()
+    assert(hasattr(raw, 'annotations') and
+           raw.annotations.orig_time is None)
```

Comment:
It should be I think;

`assert raw.annotations.orig_time == raw.info['meas_date']`